src/modules/endpoint/reducers/cityContentReducer.js

```diff
@@ -17,10 +17,18 @@ export default (
       return pushEvent(state, action)
     case 'MORPH_CONTENT_LANGUAGE':
       return morphContentLanguage(state, action)
-    case 'CLEAR_CATEGORY':
+    case 'FETCH_EVENT':
+    case 'CLEAR_EVENT': {
+      const {key} = action.params
+      delete state.eventsRouteMapping[key]
+      return state
+    }
```

Comment:
What's the use of the brackets? And why would you want to delete stuff on `FETCH_EVENT`/`FETCH_CATEGORY`?

Reply:
the brackets are used to scope the variables (case doesn't do that) and therefor I can use `key` as a new const variable in the next case.
I delete the requested route in the FETCH_EVENT/FETCH_CATEGORY to let the screen know, that sth for their screen is being loaded (only necessary if there has been something on the screen which was invalidated by the pull-to-refresh user interaction). (In the future we could also add some `loading` property to the route, but I guess I'd do that in a separate task)